content/gRPC/_index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "gRPC"
+date: 2018-08-5T14:52:17-07:00
+draft: false
+weight: 70
+class: "resized-logo"
+---
+
+![gRPC](/images/grpc.png)
+
+gRPC is a high performance, open-source universal RPC framework.
+Out of the box, OpenCensus is integrated with gRPC in the following languages:
```

Comment:
What does "out of the box" mean? I would just say that "OpenCensus provides integration with gRPC in the following languages..."